### PR TITLE
Dropdown-translation

### PR DESCRIPTION
**Pull request by ddcat1115**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [x] Rebase before creating a PR to keep commit history clear.
- [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by ddcat1115** [comment]
#1471

File: components/dropdown/demo/event.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 触发事件
+title:
+  zh-CN: 触发事件
+  en-US: Trigger event
```
Comment (benjycui) [inline components/dropdown/demo/event.md]: Events

File: components/dropdown/demo/overlay-visible.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: The way of hiding menu.
 ---
 
+## zh-CN
+
 默认是点击关闭菜单，可以关闭此功能。
 
+## en-US
+
+The default is to close the menu when you click on menu items, this feature can be turned off.
```
Comment (benjycui) [inline components/dropdown/demo/overlay-visible.md]: default behaviour?

File: components/dropdown/index.en-US.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+> Warning: You must set a unique `key` for `Menu.Item`.
+
+
+### DropdownButton
+
+| Property         | Description           | Type     | Default       |
+|--------------|----------------|----------|--------------|
+| type        | type of the button, the same as [Button](/components/button)   | String   | 'default'           |
+| onClick | a callback function, the same as [Button](/components/button), witch will be executed when you click the button on the left       | Function   | -           |
```
Comment (benjycui) [inline components/dropdown/index.en-US.md]: witch => which

**Comment by benjycui** [comment]
另外麻烦 rebase 一下哈。

File: components/dropdown/demo/overlay-visible.md (hunk 2 above)
Comment (ddcat1115) [inline components/dropdown/demo/overlay-visible.md]: seems that `behaviour ` can be omitted ?

File: components/dropdown/index.en-US.md (hunk 3 above)
Comment (ddcat1115) [inline components/dropdown/index.en-US.md]: ... 😂

File: components/dropdown/demo/overlay-visible.md (hunk 2 above)
Comment (benjycui) [inline components/dropdown/demo/overlay-visible.md]: I am not sure, see:

image

Comment (ddcat1115) [inline components/dropdown/demo/overlay-visible.md]: I think the `default` can be treated as a noun this case besides adj. ~
Also there are lots of examples:

**Comment by ddcat1115** [comment]
fixed~

**Pull request merged**